### PR TITLE
added '--defaults' as a repeatable option

### LINKED ISSUE #397: command module doesn't allow execution of multiple --defaults
pandoc's new(ish) defaults feature allows calling multiple default files from the commandline. But when I try it from :Pandoc, only one is called.

Tried: 

`:Pandoc! latex -dBase -dGS` and got:

`20210219 16:15:40 vim-pandoc:ran pandoc -s --to="latex" --output="ook.tex" --defaults="GS" "ook.md"`

### PR DESCRIPTION
**Pull request by kbro237**
Not sure if this is the cleanest code (and there may need to be other options added) but this fixed #397 for me.

### CODE REVIEW

File: python3/vim_pandoc/helpparser.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def wrap_flag(flag):
             extra = {}
             extra['action'] = 'store_true' if not opt.arg else 'store'
             # some options can be given several times
-            if any(map(lambda x: x.isupper() and x != 'T' or x == 'bibliography', opt.names)):
+            if any(map(lambda x: x.isupper() and x != 'T' or x == 'bibliography' or x == 'defaults', opt.names)):
```
Comment (fmoralesc) [inline python3/vim_pandoc/helpparser.py:119]: what about

``` python
repeatable_options = ['bibliography', 'defaults']
if any(map(lambda x: x.isupper() and x != 'T' or x in repeatable_options, opt.names)):
```

?

Comment (kbro237) [reply python3/vim_pandoc/helpparser.py]: Ok, I went thru the manual and added to your list in the next commit.

Comment (fmoralesc) [reply python3/vim_pandoc/helpparser.py]: Excellent! Thanks so much.

**Comment by fmoralesc** [comment]
LGTM. If it is ready, I will merge it.

**Pull request merged**